frontend/angular/README.md

```diff
@@ -9,6 +9,7 @@
 - [Useful Documentation](#useful-documentation)
 
 ## Project Bootstrapper
+We use [angular CLI](https://cli.angular.io/) to create an application that already works. You'll find a detailed guide to start with your project in its [README](https://github.com/Wolox/tech-guides/frontend/angular/docs/kickoff.md).
```

Comment:
`We use the angular CLI to create web applications. You'll find a detailed guide to start your project [here] (...)`